.github/workflows/lint_golang.yml

```diff
@@ -1,34 +1,33 @@
-# DONT EDIT. This file is synced from https://github.com/cloudquery/.github/.github
-name: lint_golang
+name: lint-monorepo
 on:
   push:
     branches:
-      - main
+      - monorepo
   pull_request:
     branches:
-      - main
-
+      - monorepo
```

Comment:
Shall we maybe keep these both `main` and `monorepo` for now, so it will still work when we do eventually make this the main branch? Then we can do some clean-up by removing `monorepo`, rather than having to pro-actively fix the build.